docs/shared/react-standalone-tutorial/react-standalone.md

```diff
@@ -120,65 +120,47 @@ Nx uses the following syntax to run tasks:
 
 ![Syntax for Running Tasks in Nx](/shared/images/run-target-syntax.svg)
 
-All targets, such as `serve`, `build`, `test` or your custom ones, are defined in the `project.json` file.
+## Identify Tasks
 
-```json {% fileName="project.json"}
-{
-  "name": "myreactapp",
-  ...
-  "targets": {
-    "serve": { ... },
-    "build": { ... },
-    "preview": { ... },
-    "test": { ... },
-    "lint": { ... },
-    "serve-static": { ... },
-  },
-}
+Nx identifies available tasks for your project from [tooling configuration files](/concepts/inferred-tasks), `package.json` scripts and the targets defined in `project.json`. To view the tasks that Nx has detected, look in the [Nx Console](/features/integrate-with-editors) project detail view or run:
+
+```shell
+nx show project myreactapp --web
 ```
 
-{% callout type="info" title="You can also use the package.json" %}
+If you expand the `build` task, you can see that it was created by the `@nx/vite` plugin by analyzing your `vite.config.ts` file. Notice the outputs are defined as `{projectRoot}/dist/myreactapp`. This value is being read from the `build.outDir` defined in your `vite.config.ts` file. Let's change that value in your `vite.config.ts` file:
+
+```ts {% fileName="vite.config.ts" %}
+export default defineConfig({
+  // ...
+  build: {
+    outDir: './build/myreactapp',
+    // ...
+  },
+});
+```
 
-Note that Nx can pick up tasks from both, the `package.json` as well as the `project.json`. [Read more](/features/run-tasks)
+Now if you look at the project details view, the outputs for the build target will say `{projectRoot}/build/myreactapp`. This feature ensures that Nx will always be caching the correct files.
 
-{% /callout %}
+## Overriding Inferred Tasks
```

Comment:
🤔 I'd wrap this into the "Inferred Tasks" section instead of having a separate one and keep it much shorter. Basically saying you can override the config etc & more details and link to that section in the docs.

I don't want to emphasize all of this too much even before you get started with the actual tutorial. Makes it feel convoluted otherwise I think.